universe/src/storage.rs

```diff
@@ -0,0 +1,29 @@
+use proto::universe::{KeyRange, KeyRangeRequest, KeyspaceInfo, Zone};
+use std::sync::Arc;
+use thiserror::Error;
+
+pub mod cassandra;
+
+#[derive(Clone, Debug, Error)]
+pub enum Error {
+    #[error("Timeout Error")]
+    Timeout,
+    #[error("Serialization Error: {0}")]
+    DeserializationError(Arc<dyn std::error::Error + Send + Sync>),
```

Comment:
It's not particularly helpful (other than for debugging purposes) for callers to distinguish between InternalError and DeserializationError -- DeserializationError is just another kind of internal error IMO.

Reply:
Makes sense, will remove.